create_table.lua

```diff
@@ -193,9 +198,26 @@ for _, tblname in ipairs(tables) do
         end
         local tuple = box.tuple.new(t)
 
-        box.space[tblname:upper()]:insert(tuple)
+        local rc, msg = pcall(box.space[sql_table].insert, box.space[sql_table], tuple)
+        if not rc then
+            print(('database %s failed to insert tuple %s: %s'):
+                format(sql_table, require'json'.encode(tuple), msg))
+        end
     end
     f:close()
+    box.commit()
+    -- sanity check - assume we have successfully inserted all tuples
```

Comment:
Can we throw here an error if not all tuples were inserted successfully? Also, is there a way to add tests to TPCH?